package.json

```diff
@@ -31,6 +31,7 @@
     "prepare": "node scripts/monorepo-introduction.js",
     "prettier": "prettier --loglevel warn --write '@(packages|scripts)/**/*.@(css|graphql|js)' '*.js'",
     "prettier:check": "prettier --list-different '@(packages|scripts)/**/*.@(css|graphql|js)' '*.js'",
+    "prevenia": "node ./packages/pwa-buildpack/bin/buildpack load-env ./packages/venia-concept",
```

Comment:
What is `prevenia`?

Reply:
Oh interesting. So since `venia` is just shorthand for `yarn workspace @magento/venia-concept`, this script runs every time `yarn venia` is invoked.

This is clever, but it may result in some surprising outcomes. If someone doesn't know about your `venia` shorthand, and tries to do a standard thing such as `yarn workspace @magento/venia-concept run validate-queries`, they'll miss out on `load-env` and probably be confused about why it's not working.

At least if all of that is in `venia`, they'll immediately see what the problem was.

Reply:
Fair enough. The rest of our scripts are littered with shell `&&`s, so that's what this will be too.